### PR TITLE
API: events' tags

### LINKED ISSUE #636: API: add filtering by workshop type in published events
We need to grab DC-only, or SWC-only published events. Probably there's no need to return type of the event in the structure, just filtering.

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This fixes #636 by adding:

* query param for filtering events by tags
* serializer for event tags

### CODE REVIEW

File: api/test/test_events.py [hunk 1]
```diff
@@ -144,10 +147,12 @@ def setUp(self):
                 'url': 'https://user.github.io/repository/',
                 'contact': 'sb@sth.edu',
                 'eventbrite_id': None,
+                'tags': [],
             },
         ]
 
-    @patch.object(PublishedEvents, 'request', query_params={}, create=True)
+    @patch.object(PublishedEvents, 'request', query_params=QueryDict(),
```
Comment (gvwilson) [inline api/test/test_events.py]: why this change?

File: api/views.py [hunk 2]
```diff
@@ -141,6 +148,7 @@ def get_query_params_description(self):
             'administrator': 'ID of the organization responsible for admin '
                              'work on events.',
             'host': 'ID of the organization hosting the event.',
+            'tag': 'Events\' tag. You can use this parameter multiple times.',
```
Comment (gvwilson) [inline api/views.py]: I would write as `"Events; tags(s). You can use this parameter multiple times."` (double quote to avoid escaping the single quote, then a bit of grammar)

**Comment by gvwilson** [comment]
minor question and minor suggestion, then go ahead and merge.

File: api/test/test_events.py (hunk 1 above)
Comment (pbanaszkiewicz) [inline api/test/test_events.py]: Because the view started using `query_params.getlist` which is specific to `QueryDict`, not "normal" `dict`. This change is compatible with `dict` and prevents AttributeError.
